### DISCUSSION TITLE
Can we have a new release of Leaflet with the recent fixes in main?

### DISCUSSION THREAD

**Discussion by jeffreykog** [Leaflet/Leaflet#9559] [General]
Hello! I use Leaflet in an Angular application that uses SSR. The hard references to `window` are a problem in a SSR context. But i've noticed that last summer a commit was done making all these references conditional. It would be great if we can have a 1.9.5 release containing this fix. I think nothing else would break by making these references conditional, so having this as a minor release would be fine. Thanks a lot!

https://github.com/Leaflet/Leaflet/commit/35d79e012b802b61ecf958e2b956a24aae6ef62c

**Comment by ckLocal** [comment]
I also need version 1.9.5, please release it soon as soon as possible. @jonkoops @Falke-Design @IvanSanchez
